### PR TITLE
Review notebook 7 "Reshaping"

### PR DESCRIPTION
**Pull request by jorisvandenbossche**

### CODE REVIEW

File: notebooks/7_reshape_table_layout.ipynb [hunk 1]
```diff
@@ -0,0 +1,1654 @@
+{
```
Comment (justmarkham) [inline notebooks/7_reshape_table_layout.ipynb:1]: <p>The final sentence gets lost in this paragraph, even though I think it's something you were trying to emphasize.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/stijnvanhoey/pandas-getting-started-tutorials/pull/9/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='RESOLVED' data-cellIndex='3'/>

Comment (justmarkham) [inline notebooks/7_reshape_table_layout.ipynb:1]: <p>Stating it this way makes it sound like the <code>no2</code> dataset already exists. I would rephrase it to indicate that you are filtering the <code>air_quality</code> DataFrame in order to create a new DataFrame which only contains the NO2 measurements.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/stijnvanhoey/pandas-getting-started-tutorials/pull/9/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='RESOLVED' data-cellIndex='5'/>

Comment (justmarkham) [inline notebooks/7_reshape_table_layout.ipynb:1]: <p>typo: "titanic date" should be "titanic data"</p><p> </p><p>(This same typo is also below.)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/stijnvanhoey/pandas-getting-started-tutorials/pull/9/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='RESOLVED' data-cellIndex='9'/>

Comment (justmarkham) [inline notebooks/7_reshape_table_layout.ipynb:1]: <p>I don't quite follow what the last sentence in this paragraph means. I would recommend either cutting that sentence or perhaps demonstrating it.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/stijnvanhoey/pandas-getting-started-tutorials/pull/9/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='RESOLVED' data-cellIndex='13'/>

Comment (justmarkham) [inline notebooks/7_reshape_table_layout.ipynb:1]: <p>typo: "restructering" should be "restructuring" (or you could just say "reshaping")</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/stijnvanhoey/pandas-getting-started-tutorials/pull/9/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='RESOLVED' data-cellIndex='21'/>

Comment (justmarkham) [inline notebooks/7_reshape_table_layout.ipynb:1]: <p>typo: "can to be used" should be "can be used"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/stijnvanhoey/pandas-getting-started-tutorials/pull/9/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='RESOLVED' data-cellIndex='30'/>

Comment (justmarkham) [inline notebooks/7_reshape_table_layout.ipynb:1]: <p>typo: "olumn" should be "column"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/stijnvanhoey/pandas-getting-started-tutorials/pull/9/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='RESOLVED' data-cellIndex='44'/>

Comment (stijnvanhoey) [reply notebooks/7_reshape_table_layout.ipynb:1]: <p>I'll move the filtering to the subsection where the data is effectively used instead of making it part of the introduction. </p>

Comment (stijnvanhoey) [reply notebooks/7_reshape_table_layout.ipynb:1]: <p>I'll cut this as it is not adding added value explaining all these options in detail.</p>
